src/pages/speaker-list/speaker-list.html

```diff
@@ -8,17 +8,14 @@
 </ion-header>
 
 <ion-content class="outer-content speaker-list">
-  <ion-card *ngFor="let speaker of speakers" class="speaker-card">
+  <ion-card *ngFor="let speaker of speakers" class="speaker">
 
     <ion-card-header>
       <ion-item>
         <ion-avatar item-left>
-          <img [src]="speaker.profilePic">
+          <img [src]="speaker.profilePic" (click)="goToSpeakerDetail(speaker)">
         </ion-avatar>
-        {{speaker.name}}
-        <button ion-button icon-only (click)="openContact(speaker)" clear item-right>
-          <ion-icon name="help-circle"></ion-icon>
-        </button>
+        <button ion-button clear  (click)="goToSpeakerDetail(speaker)">{{speaker.name}}</button>
```

Comment:
Maybe this could be changed to where the entire item is clickable but without the arrow?

```
    <ion-card-header>
      <button ion-item detail-none (click)="goToSpeakerDetail(speaker)">
        <ion-avatar item-left>
          <img [src]="speaker.profilePic">
        </ion-avatar>
        {{speaker.name}}
      </button>
    </ion-card-header>
```

It's a bit confusing to click the text to navigate to info about the speaker. I wasn't expecting it when I first saw it.